aws/resource_aws_launch_configuration.go

```diff
@@ -361,7 +361,7 @@ func resourceAwsLaunchConfigurationCreate(d *schema.ResourceData, meta interface
 		createLaunchConfigurationOpts.PlacementTenancy = aws.String(v.(string))
 	}
 
-	if v, ok := d.GetOk("associate_public_ip_address"); ok {
```

Comment:
As `GetOkExists` is deprecated and this has a default value. Lets just change this to `d.Get()` instead

Reply:
oops, I put my reply in the pull request. If unspecified in the config, using Get would return a setting of false, which would go in the request and override a subnet setting of "true" which is not desirable and a breaking change.